org.olivevideoeditor.Olive.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: org.olivevideoeditor.Olive
+branch: stable
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.11'
+command: olive-editor
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --env=OLIVE_EFFECTS_PATH=/app/share/olive-editor/effects
+cleanup:
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: 'x264'
```

Comment:
I don't think it has to be quoted – but it doesn't really matter.

Reply:
It makes the syntax highlighter of Github happy.